invoicing_app/tests/factories/order.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import factory
+from datetime import datetime as dt
+
+
+class OrderFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = 'invoicing_app.Order'
+        django_get_or_create = (
+            'status',
+            'pp_date',
+            'changed',
+            'event',
+            'mg_fee',
+            'gross',
+            'eb_tax'
+        )
+
+    status = 100
+    pp_date = str(dt(2020, 3, 8, 0, 0))
```

Comment:
Maybe you could use something like `factory.fuzzy.FuzzyDateTime` if you want to generate random Datetimes, also I think using str() is not necessary.